GTG/gtk/ui/general_preferences.ui

```diff
@@ -0,0 +1,390 @@
+                  <object class="GtkLabel">
+                    <property name="can_focus">False</property>
+                    <property name="xalign">0.0</property>
+                    <property name="label" translatable="yes">Start of the day</property>
+                    <attributes>
+                      <attribute name="weight" value="bold"/>
+                    </attributes>
+                  </object>
+                </child>
+                <child>
+                  <object class="GtkLabel">
+                    <property name="label" translatable="yes">Automatically refreshes the task list and adjusts WorkView</property>
+                    <property name="visible">true</property>
+                    <property name="xalign">0.0</property>
+                    <style>
+                      <class name="dim-label"/>
+                    </style>
+                    <attributes>
+                      <attribute name="scale" value="0.833333"/>
+                    </attributes>
+                  </object>
+                </child>
+              </object>
+              <packing>
+                <property name="position">1</property>
+              </packing>
+            </child>
+          </object>
+          <packing>
+            <property name="expand">False</property>
+            <property name="fill">True</property>
+          </packing>
+        </child>
+        <child>
+          <object class="GtkLabel" id="editor_label">
+            <property name="visible">true</property>
+            <property name="label" translatable="yes">Task Editor Font</property>
+            <property name="xalign">0.0</property>
+            <property name="margin-start">10</property>
+            <property name="margin-top">6</property>
+            <style>
+              <class name="dim-label"/>
+            </style>
+            <attributes>
+              <attribute name="weight" value="bold"/>
+            </attributes>
+          </object>
+        </child>
+        <child>
+          <object class="GtkFontButton" id="font_button">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="valign">center</property>
+            <property name="receives_default">False</property>
+            <property name="margin-start">40</property>
+            <property name="margin-end">40</property>
+            <property name="margin-top">10</property>
+            <signal name="font-set" handler="on_font_change" swapped="no"/>
+          </object>
+          <packing>
+            <property name="expand">False</property>
+            <property name="fill">True</property>
+          </packing>
+        </child>
+      </object>
+    </child>
+  </object>
+<!--   <packing>
+    <property name="name">general</property>
+    <property name="title" translatable="yes">General Settings</property>
+    <property name="position">0</property>
+  </packing> -->
```

Comment:
Sorry, haven't been working with general_preferences.ui for a while, must have forgotten about it